tests/Common/ConfigurationRowTest.php

```diff
@@ -30,6 +32,54 @@ public function setUp()
         }
     }
 
+    public function testConfigurationRowReturnsSingleRow()
+    {
+        $components = new Components($this->_client);
+        $configuration = new Configuration();
+        $configuration
+            ->setComponentId('wr-db')
+            ->setConfigurationId('main-1')
+            ->setName('Main');
+        $components->addConfiguration($configuration);
+
+        $configurationRow1 = new ConfigurationRow($configuration);
+        $configurationRow1->setRowId('main-1-1');
+        $components->addConfigurationRow($configurationRow1);
+
+        $configurationRow2 = new ConfigurationRow($configuration);
+        $configurationRow2->setRowId('main-1-2');
+        $components->addConfigurationRow($configurationRow2);
+
+        # qwe
+        $row = $components->getConfigurationRow(
+            (new ListConfigurationRowsOptions())
+                ->setComponentId('wr-db')
+                ->setConfigurationId('main-1')
+                ->setRowId('main-1-2')
+        );
+
+        $this->assertEquals('main-1-2', $row['id']);
+    }
+
+    public function testConfigurationRowThrowsNotFoundException()
+    {
+        $components = new Components($this->_client);
+        $configuration = new Configuration();
+        $configuration
+            ->setComponentId('wr-db')
+            ->setConfigurationId('main-1')
+            ->setName('Main');
+        $components->addConfiguration($configuration);
+
+        $this->expectException(ClientException::class);
```

Comment:
Když už se to testuje, tak by se možná hodilo i otestovat jakou to vrací message `expectExceptionMessage()`, co myslíš?